components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ReactNode, ReactElement, CSSProperties, Component, cloneElement } from 'react'
+import { BreadcrumbItemProps, StyledText, StyledActive } from './BreadcrumbItem'
+
+export interface BreadcrumbProps {
+  /** separate to be using between crumbs */
+  separator?: string | ReactNode
+  /** styles specified by the user */
+  style?: CSSProperties
+  /** Breadcrumb Items in children */
+  children?: ReactNode[] | ReactNode
+  /** className for user to specify their class */
+  className?: string
+  /** Max width of the breadcrumb to be wrapped in */
+  maxWidth?: string
+  /** Level till which the crumbs are to be shown (from end) First is shown anyways */
+  truncateTo?: number
+  /** Styles for the active item */
+  activeStyles?: CSSProperties
+}
+
+export default class Breadcrumb extends Component<BreadcrumbProps, any> {
+  static Item: React.FunctionComponent<BreadcrumbItemProps>
+  static defaultProps = {
+    separator: '/'
```

Comment:
`truncateTo` can be defaulted to `Infinity` here.

Reply:
Ahh forgot to do here and only did in function, updating 👍